renderer/index.js

```diff
@@ -211,12 +254,25 @@ function isNotTorrentFile (file) {
   return !isTorrentFile(file)
 }
 
-function addTorrent (torrentId) {
-  if (!torrentId) torrentId = 'magnet:?xt=urn:btih:6a9759bffd5c0af65319979fb7832189f4f3c35d&dn=sintel.mp4'
-  var torrent = state.client.add(torrentId)
+// Adds a torrent to the list, starts downloading/seeding
+function addTorrent (torrentOrMagnetURI) {
+  if (!torrentOrMagnetURI) torrentOrMagnetURI = 'magnet:?xt=urn:btih:6a9759bffd5c0af65319979fb7832189f4f3c35d&dn=sintel.mp4'
+  var torrent = startTorrenting(torrentOrMagnetURI)
```

Comment:
The convention that WebTorrent uses throughout the various repos is `torrentId`. There's just a lot of ways to specify a torrent.